tools/RunVersionController.ps1

```diff
@@ -272,11 +272,13 @@ function Bump-AzVersion
     $changeLog += "## $newVersion - $Release"
     foreach ($updatedModule in $updatedModules)
     {
+        $moduleMetadata = $(Get-ModuleMetadata -Module $updatedModule -RootPath $rootPath)
+        $moduleReleaseNotes = $moduleMetadata.PrivateData.PSData.ReleaseNotes
         $releaseNotes += $updatedModule
-        $releaseNotes += $(Get-ReleaseNotes -Module $updatedModule -RootPath $rootPath) + "`n"
+        $releaseNotes +=  + "`n"
```

Comment:
Correct me if I'm wrong, if we are not adding release notes to `$releaseNotes`, the "ReleaseNotes" in "Az.psd1" would be empty.